js/README.md

```diff
@@ -0,0 +1,4 @@
+Generate JSON Definiton
```

Comment:
Could you also update the results section on unknown enums (https://github.com/vchudnov-g/intjson#by-language-1)? We want to see whether the enum related classes were generated (and it looks from intjson.d.ts like indeed they were)